wiki/Table-of-Contents.md

```diff
@@ -0,0 +1,103 @@
+      }
+    }
+  ]
+}
+```
+
+Once you have reloaded Foundry to ensure the manifest changes take effect and launched into your world, you should now find that opening the compendium results in a blank page with "Contents" at the top. Perfect! That means everything is working so far, but you will need to set some additional flags to indicate to the system what journal entries should appear in the table of contents and in what order.
+
+![Empty table of contents page](https://raw.githubusercontent.com/foundryvtt/dnd5e/publish-wiki/wiki/images/journal/table-of-contents-empty.jpg)
+
+### Setting Journal Entry Flags
+
+By default no journal entries are displayed in the table of contents. There are at least two flags that must be set on a journal entry to make it appear in the correct position:
+
+#### `type` flag
+
+The `type` flag indicates how the journal entry will be sorted and displayed on the ToC. There are four types supported by the system:
+- `chapter`: Chapters are sorted first and the entry name is displayed as a large heading. They will automatically list all of their pages below.
+- `appendix`: Appendices are sorted last and also displayed as a large header. By default they do not display their individual pages, but that can be overridden using the `showPages` flag (see below).
+- `special`: Special is used for journal entries that should appear as if they are pages beneath a chapter or appendix. They also hide their pages by default.
+- `header`: The first page of this journal entry will be displayed inline at the top of the table of contents.
+
+```javascript
+const journalEntry = await fromUuid("...");
+journalEntry.setFlag("dnd5e", "type", "chapter");
+```
+
+#### `showPages` flag
+
+The `showPages` flag on entries controls whether the individual pages are listed beneath the entry name in the list with their own links. This defaults to `true` for entries with the `chapter` type, and `false` for entries with the `appendix` or `special` types.
+
+```javascript
+const journalEntry = await fromUuid("...");
+journalEntry.setFlag("dnd5e", "showPages", false);
+```
+
+#### `position` flag
+
+The `position` flag is used to control where chapters and appendices appear in the list. It is usually as simple as setting the first chapter to have a `1`, the second chapter to have `2`, and so on. The position is independent between chapters and appendices, so you can have appendix A also have an position of `1`.
+
+```javascript
+const journalEntry = await fromUuid("...");
+journalEntry.setFlag("dnd5e", "position", 1);
+```
+
+This flag only has effect for `chapter` and `appendix` entries.
+
+#### `append` & `order` flags
+
+The `append` and `order` flags are used by `special` entries to control where they appear in the final list. The `append` flag specifies which chapter the special entry will be added to. If no `append` is provided, then a special entry will be added after all of the other chapters and appendices as a top-level entry.
```

Comment:
```suggestion
The `append` and `order` flags are used by `special` entries to control where they appear in the final list. The `append` flag specifies to which chapter the special entry will be added. If no `append` is provided, then a special entry will be added after all of the other chapters and appendices as a top-level entry.
```

Reply:
English grammar does not forbid ending sentences with prepositions. The original construction was better.